packages/backend/src/execution/executors/PgExecutor/PgExecutor.ts

```diff
@@ -32,6 +32,8 @@ interface PgExecutorProps {
     logging?: boolean;
     /**
      * An SQL statement that will be sent before every SynthQL query.
+     *
+     * e.g `SET search_path TO "public";`
```

Comment:
Is this a good example? I think messing with the search path can results in bugs. We make some assumptions about your search path being at public.

Reply:
Changed it to `SELECT version();`